samples/bluetooth/central_hids/src/main.c

```diff
@@ -97,7 +97,7 @@ static void scan_connecting(struct bt_scan_device_info *device_info,
 {
 	default_conn = bt_conn_ref(conn);
 }
-
+/** .. include_startingpoint_scan_rst */
```

Comment:
How does this work exactly @KAGA164 ?

Reply:
It's me who added this, @carlescufi . This is to avoid duplicating code when a snippet is included in an .rst file. You enclose the lines with tags, and in the .rst specify the file and the tags. See edits to `scan.rst` for breakdown.

Reply:
Redirecting this question to @greg-fer who made this comments.

Reply:
https://www.sphinx-doc.org/en/1.5/markup/code.html#directive-literalinclude